### PR TITLE
Update embulk-util-timestamp to 0.2.1

### PR DESCRIPTION
**Pull request by dmikurube**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -28,28 +28,29 @@ repositories {
 }
 
 dependencies {
-    compileOnly("org.embulk:embulk-api:0.10.5") {
+    compileOnly("org.embulk:embulk-api:0.10.12") {
         exclude group: "*"
     }
-    compileOnly("org.embulk:embulk-core:0.10.5") {
+    compileOnly("org.embulk:embulk-api:0.10.12") {
```
Comment (luongnhattruong) [inline build.gradle]: i think it is redundant.

**Review by luongnhattruong — APPROVED** [review]
LGTM. Except minor comment

File: build.gradle (hunk 1 above)
Comment (dmikurube) [reply build.gradle]: Ah, thanks for the good catch! I intended this to be `embulk-Spi`, but I mistook... Fixed in: 204fdd95870e05152f91ce896894654a291fcff1

**Comment by dmikurube** [comment]
Thanks! I think 204fdd95870e05152f91ce896894654a291fcff1 and 1cb12ecb338cbaab7dc3647dbdf21ca1a3406af8 are trivial. Merging and releasing.

**Pull request merged**